backend/core/tests/test_api_views.py

```diff
@@ -124,3 +125,99 @@ def test_staff_users_cant_see_all_teams(self):
         url = self.get_list_url()
         response = self.client.get(url)
         self.assertEqual(len(response.data), 0)
+
+
+class TestBulkSetDueDatesApi(APITestCase, APITestCaseMixin):
```

Comment:
Is there a reason this isn't in **curriculum_tracking/tests**?

Reply:
In the instructions of this ticket it says this at the bottom "Tests can go in core/tests/test_api_views.py"